cypress.json

```diff
@@ -1,5 +1,5 @@
 {
-    "baseUrl": "http://127.0.0.1:8080",
+    "baseUrl": "https://urecha.github.io/",
```

Comment:
This shouldn't be pointing to your personal instance anymore

Reply:
Set back to localhost.